doc/source/getting_started/docker.rst

```diff
@@ -43,39 +45,80 @@ with:
     cat GH_TOKEN.txt | docker login ghcr.io -u $GH_USERNAME --password-stdin
 
 
+.. _run_an_mapdl_image:
+
 Run an MAPDL image
 ===================
 
 You can now launch MAPDL directly from Docker with a short script or
-directly from the command line. Because this image contains no license
-server, you must enter your license server IP address in the
-``LICENSE_SERVER`` environment variable. With that, you can launch
-MAPDL with:
-
-.. code::
+directly from the command line.
+You can use then your host Python environment to connect to that MAPDL
+instance.
+
+.. graphviz::
+
+    digraph {
+      node [];
+      subgraph cluster_frontend {
+        label="*HOST*";
+        Python;
+      subgraph cluster_backend {
+        MAPDL;
+        label="*DOCKER*";
+        }
+      }
+      Python -> MAPDL
+      MAPDL -> Python
+    }
+
+Because this image contains no license server, you must enter your
+license server IP address in the ``ANSYSLMD_LICENSE_FILE`` environment variable. 
+
+Additionally, you run a Docker image of PyMAPDL with:
 
-    LICENSE_SERVER=1055@XXX.XXX.XXX.XXX
-    VERSION=v21.1.0
+.. code:: pwsh
 
-    IMAGE=ghcr.io/pyansys/pymapdl/mapdl:$VERSION
-    docker run -e ANSYSLMD_LICENSE_FILE=$LICENSE_SERVER -p 50052:50052 $IMAGE -smp
+    docker run -e ANSYSLMD_LICENSE_FILE=1055@host.docker.internal --restart always --name mapdl -p 50053:50052 ghcr.io/pyansys/pymapdl/mapdl -smp > log.txt
 
 First time you run it, Docker logins into the *ghcr.io* registry and
 pulls the image which can take some time.
 There are several images in the registry, each corresponding to a different
 version of MAPDL. It is recommended to use the latest version of MAPDL for
 the Ubuntu systems (any image tagged as ``Ubuntu``).
 
-Note that port `50052` (local to the container) is being mapped to
-50052 on the host. This makes it possible to launch several MAPDL
-instances with different port mappings to allow for multiple instances
-of MAPDL.
+Successive runs should restart the container or just delete it and rerun it using:
```

Comment:
```suggestion
To rerun it again, you should restart the container or just delete it and run it again using:
```

Reply:
Better like this?? @PipKat

Reply:
@germa89 Almost--"rerun" means run again. So either "To rerun, you should...." or "To run again, you should...."